ibu-imager/ops/ops.go

```diff
@@ -153,6 +153,11 @@ func (o *ops) RunRecert(recertContainerImage, authFile string, additionalArgs ..
 		"--static-dir", "/kubelet",
 		"--static-dir", "/machine-config-daemon")
 
+	// Create BackupCertsDir
+	if err := os.MkdirAll(common.BackupCertsDir, 0o700); err != nil {
```

Comment:
I'd add it in [seedcreator.go#L81](https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/seedcreator/seedcreator.go#L81), just after the creation of `BackupChecksDir` and before `createContainerList`.

Reply:
in reality, this volume is not been used yet (can be removed)... just spotted this has also been fixed [here](https://github.com/openshift-kni/lifecycle-agent/pull/101/files#diff-86bb331a0505140ae7958d76789e7803cf3d42f4700e8db04a609f58a05f52c3L148) at #101.